static/js/interpreter.js

```diff
@@ -1,8 +1,12 @@
 import {execute} from "./execution.js";
 
 const introText =
-`Welcome to the 61A SQL interpreter!
-Enter some SQL below:
+`61A SQL Web Interpreter
+<a href="https://github.com/Cal-CS-61A-Staff/queryable">on GitHub</a>
+--------------------------------------------------------------------------------
+Welcome to the 61A SQL interpreter!
+Type <code>.help</code> for instructions, or <code>.read</code> to load a file.
```

Comment:
load a file from where?

Reply:
It opens a file upload dialog.

Reply:
very unintuitive error messages

![image](https://user-images.githubusercontent.com/3214932/56453022-b1ba6780-62ef-11e9-998d-e4a753afcb54.png)

Reply:
I think say run "run `.read` to select a file from your computer"